frontend/ui/widget/infomessage.lua

```diff
@@ -1,19 +1,21 @@
-local InputContainer = require("ui/widget/container/inputcontainer")
+local Blitbuffer = require("ffi/blitbuffer")
 local CenterContainer = require("ui/widget/container/centercontainer")
-local Font = require("ui/font")
 local Device = require("device")
-local GestureRange = require("ui/gesturerange")
+local Font = require("ui/font")
 local FrameContainer = require("ui/widget/container/framecontainer")
+local Geom = require("ui/geometry")
+local GestureRange = require("ui/gesturerange")
 local HorizontalGroup = require("ui/widget/horizontalgroup")
+local HorizontalSpan = require("ui/widget/horizontalspan")
 local ImageWidget = require("ui/widget/imagewidget")
+local InputContainer = require("ui/widget/container/inputcontainer")
+local ScrollTextWidget = require("ui/widget/scrolltextwidget")
 local TextBoxWidget = require("ui/widget/textboxwidget")
-local HorizontalSpan = require("ui/widget/horizontalspan")
 local UIManager = require("ui/uimanager")
-local Geom = require("ui/geometry")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local _ = require("gettext")
 local Input = require("device").input
 local Screen = require("device").screen
-local _ = require("gettext")
-local Blitbuffer = require("ffi/blitbuffer")
 
 --[[
 Widget that displays an informational message
```

Comment:
If you have the time it could be neat to add some docs akin to https://github.com/koreader/koreader/blob/1b700c00443da111bdd3eb0bad6a42a9a8f2a370/frontend/ui/widget/inputdialog.lua#L1-L40